docs/source/guide/index.rst

```diff
@@ -380,6 +380,342 @@ Using another computer with access to ``repository-service-tuf-api``
+    ┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+    │ 800dfb5a1982b82b7893e58035e19f414f553fc08cbb1130cfbae302a7b7fee5 │ Steven's Key │ ed25519  │ Offline │ 7098f769f6ab8502b50f3b58686b8a042d5d3bb75d8b3a48a2fcbc15a0223501 │
+    │ 1cebe343e35f0213f6136758e6c3a8f8e1f9eeb7e47a07d5cb336462ed31dcb7 │ Kairo's Key  │ ed25519  │ Offline │ ad1709b3cb419b99c5cd7427d6411522e5a93aec6767453e91af921a73d22a3c │
+    └──────────────────────────────────────────────────────────────────┴──────────────┴──────────┴─────────┴──────────────────────────────────────────────────────────────────┘
+
+    Do you want to add a new key? [y/n]: n
+
+    Here is the current content of root:
+
+
+    ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+    ┃ Root                         ┃                                                                                    KEYS                                                                                     ┃
+    ┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+    │                              │                                                                    ╷              ╷          ╷         ╷                                                                    │
+    │                              │                                  Id                                │   Name/Tag   │ Key Type │ Storage │                           Public Value                             │
+    │ Number of Keys: 2            │ ╶──────────────────────────────────────────────────────────────────┼──────────────┼──────────┼─────────┼──────────────────────────────────────────────────────────────────╴ │
+    │ Threshold: 1                 │   800dfb5a1982b82b7893e58035e19f414f553fc08cbb1130cfbae302a7b7fee5 │ Steven's Key │ ed25519  │ Offline │ 7098f769f6ab8502b50f3b58686b8a042d5d3bb75d8b3a48a2fcbc15a0223501   │
+    │ Root Expiration: 2024-May-28 │   1cebe343e35f0213f6136758e6c3a8f8e1f9eeb7e47a07d5cb336462ed31dcb7 │ Kairo's Key  │ ed25519  │ Offline │ ad1709b3cb419b99c5cd7427d6411522e5a93aec6767453e91af921a73d22a3c   │
+    │                              │                                                                    ╵              ╵          ╵         ╵                                                                    │
+    └──────────────────────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

Comment:
When you remove I key I immediately show you how the root keys look.
Same if you add a key.
It's hard to believe that you deleted the wrong key if you have used key names (which I think many people will use) further it's even harder to make a mistake adding a key when we ask you about this new key path, password and type.
If for whatever reason you see there is a problem I give the opportunity to modify the root keys yet again.

Maybe the most inconvenient is that if you remove a key it could be hard to add it again (if you did a mistake) if you don't have access to that key.
In this case you can always abort the operation, but if we decide it's still a risk we can ask the user for confirmation.
Still, if the user deletes a key that doesn't have a name, then even if we show him the key id of the key for him it will be nonsensical and that confirmation doesn't bring a lot of value.

Finally, it's hard to make a mistake and delete a key with a name as we ask the user to write down the name of the key for deletion.

@kairoaraujo any opinions?

Reply:
I think what you @MVrachev sounds good for this version.
If we still think we can improve the UX we could file an issue.

We know that the CLI there is some limitations as well.
I think some interactions will also change with the possibility of distributed asynchronous signing.

Reply:
@lukpueh, please do it. ❤️